index.html

```diff
@@ -10,16 +10,24 @@
 <body>
     <header>
         <a href="https://github.com/Elisha-Tetteyfio/Portfolio-setup-mobile-version-skeleton" class="menu-element" id="name">Elisha</a>
-        <nav>
+        <nav class="mobile">
             <img src="Icons/Union.svg" alt="Menu" class="menu-element" id="menu">
         </nav>
+        <div class="desktop">
+            <ul>
+                <li> <a href="#Portfolio">Portfolio</a> </li>
+                <li> <a href="#About">#About</a> </li>
+                <li> <a href="#Contact">Contact</a> </li>
+                <li><img src="Icons/Mail.svg" alt="mail"></li>
+            </ul>
+        </div>
```

Comment:
- [ ] [Optional] Your toolbar seems to be too central and the distance between the `h1` and the toolbar is too short compared to the Figma design as you can see below.
> Yours 
![image](https://user-images.githubusercontent.com/90556221/166121446-75c99367-5b33-4a96-a2e0-f1234da8a344.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/166121589-f23ec228-a39f-4c62-822c-ff19754d6bce.png)

Kindly do fix it if you have the opportunity :)